docs/paper/dev/api/how-do-plugins-work.md

```diff
@@ -0,0 +1,100 @@
+---
+slug: /dev/how-do-plugins-work
+description: How plugins work in Paper.
+---
+
+# How Plugins Work
+
+## Introduction
+
+Plugins are a way to extend the functionality of a Minecraft server. They are written in Java and can be loaded and 
```

Comment:
they can be written in other languages that run on the JVM as well, such as Kotlin, Groovy or Scala